includes/container-service-faq.md

```diff
@@ -92,6 +92,16 @@ ssh userName@masterFQDN –A –p 22 
 
 For more information, see [Connect to an Azure Container Service cluster](../articles/container-service/kubernetes/container-service-connect.md).
 
+### My DNS name resolution isn't working. What should I do?
+
+There are some known DNS issues whose fixes are still actively being phased out. Please ensure you are using the most updated acs-engine and Windows version (with [KB4074588](https://www.catalog.update.microsoft.com/Search.aspx?q=KB4074588) and [KB4089848](https://www.catalog.update.microsoft.com/Search.aspx?q=KB4089848) installed) so that your environment can benefit from this. Otherwise, see the table below for mitigation steps in:
+
+| DNS Symptom | Workaround  |
+|-------------|-------------|
+|When workload container is unstable and crashes, the network namespace is cleaned up | Redeploy any affected services |
+| Service VIP access is broken | Configure a [DaemonSet](https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/) to always keep one normal (non-privileged) pod running |
+|When node on which container is running becomes unavailable, DNS queries may fail resulting in a "negative cache entry" | Run the following inside affected containers: <ul><li> `New-ItemProperty -Path 'HKLM:\SYSTEM\CurrentControlSet\Services\Dnscache\Parameters' -Name MaxCacheTtl -Value 0 -Type DWord`</li><li>`New-ItemProperty -Path 'HKLM:\SYSTEM\CurrentControlSet\Services\Dnscache\Parameters' -Name MaxNegativeCacheTtl -Value 0 -Type DWord`</li><li>`Restart-Service dnscache` </li></ul><br> If this still doesn't resolve the problem, then try to disable DNS caching completely: <ul><li>`Set-Service dnscache -StartupType disabled`</li><li>`Stop-Service dnscache`</li></ul> |
```

Comment:
We may get some push back on the inline HTLM. I am ok with trying to get this published, but be aware we may have to update to markdown.

Reply:
Understandable, it is harder to read/maintain. It currently looks like this
![image](https://user-images.githubusercontent.com/36676176/38822311-e5e95792-4157-11e8-949e-509f7deb8e2c.png)